src/styles/components/framework-configuration/styles.less

```diff
@@ -29,3 +29,7 @@
     z-index: 0;
   }
 }
+
+.glyphicon-plus:before {
```

Comment:
This element is a `<i>`, so the browser styles it as italic.

Could you add `font-style: normal` to these styles?

Reply:
The plus is not _italic_ in the screenshot, is it italic for you?

Reply:
@GeorgiSTodorov - yes, it was _italic_ for me in Chrome